src/app/CommandHandler.cpp

```diff
@@ -522,6 +524,7 @@ CHIP_ERROR CommandHandler::FinishCommand(bool aStartDataStruct)
 CHIP_ERROR CommandHandler::PrepareStatus(const ConcreteCommandPath & aCommandPath)
 {
     ReturnErrorOnFailure(AllocateBuffer());
+    mInvokeResponseBuilder.Checkpoint(mBackupWriter);
```

Comment:
But nothing in AddStatus rolls back, right?  Why are we checkpointing here?

Reply:
Yes, no need to add a checkpoint here as long as the Rollback is a private function.
I added this to make PrepareStatus and PrepareCommand a bit more consistent.